### ISSUE TITLE
python manager.py start

### ISSUE THREAD

**Issue by Nebemires** [swar/Swar-Chia-Plot-Manager#199]
Cannot start manager.py

```
Traceback (most recent call last):
  File "C:\Users\Chia 1\Swar-Chia-Plot-Manager-main\stateless-manager.py", line 58, in <module>
    jobs, running_work, next_job_work, next_log_check = monitor_jobs_to_start(
  File "C:\Users\Chia 1\Swar-Chia-Plot-Manager-main\plotmanager\library\utilities\jobs.py", line 119, in monitor_jobs_to_start
    job, work = start_work(job=job, chia_location=chia_location, log_directory=log_directory)
  File "C:\Users\Chia 1\Swar-Chia-Plot-Manager-main\plotmanager\library\utilities\jobs.py", line 169, in start_work
    process = start_process(args=plot_command, log_file=log_file)
  File "C:\Users\Chia 1\Swar-Chia-Plot-Manager-main\plotmanager\library\utilities\processes.py", line 276, in start_process
    process = subprocess.Popen(
  File "C:\Python39\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Python39\lib\subprocess.py", line 1420, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified
```

**Comment by nickbeagley** [comment]
I had the identical error message and it is potentially the same issue. this was caused by my plotter moving during upgrade from 1.1.3 to 1.1.4.  It's location is specified at the start of the yaml.

chia_location: C:\Users\USERNAME\AppData\Local\chia-blockchain\app-1.1.4\resources\app.asar.unpacked\daemon\chia.exe

If Swar checked that this location actually existed it could make a much nicer error message telling us that we had messed up our config.yaml.

In my case, i had a separate folder like people recommend for my plotter so i'm not impacted by version changes but it was within the chia-blockchain folder, so was blown away by the 1.1.3->1.1.4 upgrade.
